feat: add @kaname-png/plugin-env (#5)
sapphiredev/awesome-sapphire: README.md

## Contents

- [Awesome Sapphire ![Awesome](https://awesome.re)](#awesome-sapphire-)
  - [Contents](#contents)
  - [Plugins](#plugins)
  - [Utilities](#utilities)
  - [Reading](#reading)
  - [Miscellaneous](#miscellaneous)

>   - [Contribute](#contribute)

Reviewer: Revert

Reviewer: You still need to do this

## Plugins

- [@devtomio/plugin-botlist](https://github.com/devtomio/sapphire-plugin-botlist#readme): This plugin enables the
  integration of many bot lists such as [`top.gg`](https://top.gg), [`discordlabs`](https://discordlabs.org), and
  others.

> - [@kaname-png/plugin-env](https://github.com/kaname-png/neko-plugins/tree/main/packages/env): Manage environment
>     variables with @sapphire/framework for the bot client.

Reviewer: ```suggestion
- [@kaname-png/plugin-env](https://github.com/kaname-png/neko-plugins/tree/main/packages/env): Manage environment variables for use with any [`@sapphire/framework`](https://github.com/sapphiredev/framework) bot project.
```

And run prettier afterwards
